view/theme/frio/js/theme.js

```diff
@@ -317,17 +317,19 @@ $(document).ready(function(){
 	});
 
 	/*
-	 * Sticky aside on page scroll
+	 * Sticky aside on page scroll, only on desktop
```

Comment:
Can you please write a note why we have choosen `only on desktop`. It is really helpful after some time, when someone does change somewhere else a behavior and don't find the the source why it isn't working